rclpy/src/rclpy/_rclpy_pybind11.cpp

```diff
@@ -14,15 +14,37 @@
 
 #include <pybind11/pybind11.h>
 
+#include "clock.hpp"
 #include "context.hpp"
 #include "rclpy_common/exceptions.hpp"
 #include "service_info.hpp"
+#include "time_point.hpp"
 
 namespace py = pybind11;
 
 PYBIND11_MODULE(_rclpy_pybind11, m) {
   m.doc() = "ROS 2 Python client library.";
 
+  py::enum_<rcl_clock_type_t>(m, "ClockType")
+  .value("UNINITIALIZED", RCL_CLOCK_UNINITIALIZED)
+  .value("ROS_TIME", RCL_ROS_TIME)
+  .value("SYSTEM_TIME", RCL_SYSTEM_TIME)
+  .value("STEADY_TIME", RCL_STEADY_TIME);
+
+  py::enum_<rcl_clock_change_t>(m, "ClockChange")
+  .value(
+    "ROS_TIME_NO_CHANGE", RCL_ROS_TIME_NO_CHANGE,
+    "ROS time is active and will continue to be active")
+  .value(
+    "ROS_TIME_ACTIVATED", RCL_ROS_TIME_ACTIVATED,
+    "ROS time is being activated")
+  .value(
+    "ROS_TIME_DEACTIVATED", RCL_ROS_TIME_DEACTIVATED,
+    "ROS TIME is being deactivated, the clock will report system time after the jump")
+  .value(
+    "SYSTEM_TIME_NO_CHANGE ", RCL_SYSTEM_TIME_NO_CHANGE,
```

Comment:
There is an excess space here:
```suggestion
    "SYSTEM_TIME_NO_CHANGE", RCL_SYSTEM_TIME_NO_CHANGE,
```

Reply:
Good catch! https://github.com/ros2/rclpy/pull/699/commits/2560af0471d8635c6620c95fe5e97735f4c7d731